shims/spark311/pom.xml

```diff
@@ -30,6 +30,44 @@
     <description>The RAPIDS SQL plugin for Apache Spark 3.1.1 Shim</description>
     <version>21.10.0-SNAPSHOT</version>
 
+    <profiles>
+        <profile>
+            <id>default</id>
+            <activation>
+                <property>
```

Comment:
this definitely needs a comment, its really weird to include source from different module, I assume this is just temporary to not break current build method